lapis/cli/simulate.py

```diff
@@ -168,43 +319,128 @@ def dynamic(ctx, job_file, pool_file):
     "job_file",
     type=(click.File("r"), click.Choice(list(job_import_mapper.keys()))),
 )
+@click.option("--pre-job-rank", "pre_job_rank", type=str, default=None)
+@click.option("--machine-ads", "machine_ads", type=str, default=None)
+@click.option("--job-ads", "job_ads", type=str, default=None)
 @click.option(
-    "--static-pool-file",
-    "static_pool_file",
+    "--scheduler-type",
+    "scheduler_type",
+    type=click.Choice(list(scheduler_import_mapper.keys())),
+)
+@click.option(
+    "--static-pool-files",
+    "static_pool_files",
     type=(click.File("r"), click.Choice(list(pool_import_mapper.keys()))),
     multiple=True,
+    help="Tuple of `(static_pool_file,static_pool_file_type)`",
 )
 @click.option(
-    "--dynamic-pool-file",
-    "dynamic_pool_file",
+    "--dynamic-pool-files",
+    "dynamic_pool_files",
     type=(click.File("r"), click.Choice(list(pool_import_mapper.keys()))),
     multiple=True,
+    help="Tuple of `(dynamic_pool_file,dynamic_pool_file_type)`",
+)
+@click.option(
+    "--storage-files",
+    "storage_files",
+    type=(
+        click.File("r"),
+        click.File("r"),
+        click.Choice(list(storage_import_mapper.keys())),
+    ),
+    default=(None, None, None),
+    multiple=True,
+    help="Tuple of `(storage_file,storage_content_file,storage_type)`",
+)
+@click.option(
+    "--remote-throughput",
+    "remote_throughput",
+    type=float,
+    default=10,
+    help="Parameter to set the network bandwidth to remote",
 )
+@click.option(
+    "--filebased_caching",
+    "filebased_caching",
+    is_flag=True,
+    help="Flag to set filebased caching on/off",
+    default=False,
+)
+@click.option("--cache-hitrate", "cache_hitrate", type=float, default=None)
 @click.pass_context
-def hybrid(ctx, job_file, static_pool_file, dynamic_pool_file):
+def hybrid(
+    ctx,
+    job_file,
+    pre_job_rank,
+    machine_ads,
+    job_ads,
+    scheduler_type,
+    static_pool_files,
+    dynamic_pool_files,
+    storage_files,
+    remote_throughput,
+    filebased_caching,
+    cache_hitrate,
+):
     click.echo("starting hybrid environment")
     simulator = Simulator(seed=ctx.obj["seed"])
-    file, file_type = job_file
+    infile, file_type = job_file
     simulator.create_job_generator(
-        job_input=file,
+        job_input=infile,
         job_reader=partial(
             job_import_mapper[file_type],
             calculation_efficiency=ctx.obj["calculation_efficiency"],
         ),
     )
-    simulator.create_scheduler(scheduler_type=CondorJobScheduler)
-    for current_pool in static_pool_file:
-        file, file_type = current_pool
+    click.echo(
```

Comment:
More duplicated code following here?

Reply:
see above

Reply:
I absorbed the code in a helper function.